### ISSUE TITLE
Yolo-v4 loss is higher than yolo-v3 loss on my custom dataset.

### ISSUE THREAD

**Issue by SonaHarutyunyan** [AlexeyAB/darknet#6175]
Hi, everyone!
I am training my custom dataset with yolov3.cfg and yolov4.cfg files.
I use the same executable file compiled with GPU=1, CUDNN=1, CUDNN_HALF=1 parameters in MakeFile. I downloaded the pretrained weights (darknet53.conv.74 for yolov3) and yolov4.conv.137 (for yolov4) as it was described in README. So, for setting the trainings I run the following commands for yolov3 and yolov4 correspondingly:

./darknet detector train data/obj.data cfg/yolo-obj_3.cfg darknet53.conv.74
./darknet detector train data/obj.data cfg/yolo-obj_4.cfg yolov4.conv.137

Here yolo-obj_3.cfg is the modified copy of yolov3.cfg and cfg/yolo-obj_4.cfg is the modified copy of yolov4.cfg. I modified the number of classes in 3 places (classes = 10) and the previous layer filter number setting filters = 45 (by the given formula: (10+5)*3) again in 3 places. Also I set width = 416, height = 416, batch=64, subdivisions=32, max_batches = 20000, steps=16000,18000 in both files and I get the following loss functions:
![Screenshot from 2020-07-08 17-18-29](https://user-images.githubusercontent.com/28267003/86925331-ccba0700-c141-11ea-9200-60f68e2a4c13.png)
![Screenshot from 2020-07-08 17-19-46](https://user-images.githubusercontent.com/28267003/86925345-d17ebb00-c141-11ea-998b-7afff0e47991.png)

Although the training for yolov3 is still in progress, but I can see that it has a lower loss than the one that uses yolov4 configs. So, can you help me finding what I'm missing?
Thank you!

**Comment by kossolax** [comment]
Higher loss is not a problem, only mAP matter.
